lib/pbench/test/unit/server/query_apis/test_datasets_contents.py

```diff
@@ -392,9 +395,10 @@ def test_empty_query(
             status=HTTPStatus.OK,
             headers=build_auth_header["header"],
         )
-        res_json = response.json
-        expected_result = {"directories": [], "files": []}
-        assert expected_result == res_json
+        if expected_status == HTTPStatus.OK:
+            res_json = response.json
+            expected_result = ["NOT_FOUND"]
+            assert expected_result == res_json
```

Comment:
`expected_status` is `HTTPStatus.NOT_FOUND`, and the `response.json` should be `{"message": ...}` if we're going to bother checking it. (We do in some places, for example `test_datasets_list.py`'s `test_get_bad_keys`.)